paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -33,7 +34,7 @@ import javax.inject.Inject
 import javax.inject.Provider
 
 @OptIn(ExperimentalCustomerSheetApi::class)
-@CustomerSessionScope
+@CustomerSheetScope
```

Comment:
This isn't quite the CustomerSheetScope 🤔

Reply:
How about `CustomerSheetActivityScope`

Reply:
I think we should just call it `CustomerSheetViewModelScope` 🤷  The activity has the same lifecycle as the CustomerSheetScope